src/main/java/com/workoutbuilder/enterprise/controller/WorkoutBuilderController.java

```diff
@@ -205,9 +209,9 @@ public ResponseEntity<List<CalendarWorkout>> fetchWorkouts(
 
         List<Workout> workouts = workoutService.findAll();
 
-        var calendarWorkouts = new ArrayList<CalendarWorkout>();
+        List<CalendarWorkout> calendarWorkouts = new ArrayList<CalendarWorkout>();
```

Comment:
I replaced every 'var' keyword in this method with object names. I thought this would make it quicker to interpret the variable type when looking over the code.

This website explains the importance of readable code: https://www.baeldung.com/java-clean-code